cmd/flux/bootstrap_git.go

```diff
@@ -107,9 +107,12 @@ func bootstrapGitCmdRun(cmd *cobra.Command, args []string) error {
 	}
 
 	// Manifest base
-	if ver, err := getVersion(bootstrapArgs.version); err == nil {
+	if ver, err := getVersion(bootstrapArgs.version); err != nil {
```

Comment:
We can't return an error if `getVersion` fills as it will break Flux on air-gapped systems, flux install and bootstrap should make work even if external calls to e.g. GitHub fail.